README.rst

```diff
@@ -15,30 +15,43 @@ learning, automatic differentiation, and optimization of hybrid quantum-classica
 Installation
 ============
 
+We suggest first installing Kokkos with the wanted configuration following the instruction found in the `Kokkos documentation <https://kokkos.github.io/kokkos-core-wiki/building.html>`_.
+Next, append the install location to ``CMAKE_PREFIX_PATH``.
+If an installation is not found, our builder will install it from scratch nevertheless.
+
+The simplest way to install PennyLane-Lightning-Kokkos (OpenMP backend) is using ``pip``.
+
 .. code-block:: console
 
-   cmake -Bbuild -DKokkos_ENABLE_OPENMP=On -DPLKOKKOS_BUILD_TESTS=On -G Ninja
-   cmake --build ./build
+   CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" python -m pip install .
 
-You can install the python interface with:
+or for an editable ``pip`` installation with:
 
 .. code-block:: console
 
-   python setup.py build_ext --backend="OPENMP"
+   CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" python -m pip install -e .
+
+Alternatively, you can install the python interface with:
+
+.. code-block:: console
+
+   CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" python setup.py build_ext
    python setup.py bdist_wheel
    pip install ./dist/PennyLane*.whl --force-reinstall
 
-
-or for an editable `pip` installation with:
+To build the plugin directly with CMake:
 
 .. code-block:: console
 
-   BACKEND="OPENMP" python -m pip install -e .
+   cmake -B build -DKokkos_ENABLE_OPENMP=ON -DPLKOKKOS_BUILD_TESTS=ON -G Ninja
+   cmake --build build
 
-
-Supported backend options are "SERIAL", "OPENMP", "THREADS", "HIP" and "CUDA". For "HIP" and "CUDA", the appropriate software stacks are required to enable compilation and subsequent use.
-For explicit targeting of a given supported architecture, the environment variable `ARCH` can also be specified which directly sets the `-DKokkos_ARCH_{...}=ON` build option. Note that "THREADS"
-backend is not recommended since `Kokkos <https://github.com/kokkos/kokkos-core-wiki/blob/17f08a6483937c26e14ec3c93a2aa40e4ce081ce/docs/source/ProgrammingGuide/Initialization.md?plain=1#L67>`_ does not guarantee its safety.
+Supported backend options are "SERIAL", "OPENMP", "THREADS", "HIP" and "CUDA" and the corresponding build switches are ``-DKokkos_ENABLE_BACKEND=ON``, where one needs to replace ``BACKEND``.
+One can activate simultaneously one serial, one parallel host and one parallel host backend, but not two of any category at the same time.
+For "HIP" and "CUDA", the appropriate software stacks are required to enable compilation and subsequent use.
+Similarly, the CMake option ``-DKokkos_ARCH_{...}=ON`` must also be specified to target a given architecture.
```

Comment:
I'm curious, what is the difference between ` ``-DKokkos_ARCH_{...}=ON`` ` and `` `-DKokkos_ARCH_{...}=ON` ``?

Reply:
The first one is for RST monospaced font, the second one is for MD monospaced font (AFAIK).

Reply:
Any special reason for you changing that? I'm really just curious.

Reply:
Since it is an RST document, if we want to render these string as `inline literal`'s, we need double ``.